go.mod

```diff
@@ -1,33 +1,33 @@
 module github.com/ethersphere/bee
 
-go 1.14
+go 1.13
 
 require (
 	github.com/btcsuite/btcd v0.20.1-beta
 	github.com/coreos/go-semver v0.3.0
-	github.com/ethereum/go-ethereum v1.9.12
+	github.com/dgraph-io/badger/v2 v2.0.3
 	github.com/ethersphere/swarm v0.5.7
 	github.com/gogo/protobuf v1.3.1
 	github.com/gorilla/handlers v1.4.2
-	github.com/gorilla/mux v1.7.3
-	github.com/libp2p/go-libp2p v0.5.1
+	github.com/gorilla/mux v1.7.4
+	github.com/libp2p/go-libp2p v0.7.4
 	github.com/libp2p/go-libp2p-autonat-svc v0.1.0
-	github.com/libp2p/go-libp2p-core v0.3.0
-	github.com/libp2p/go-libp2p-peerstore v0.1.4
-	github.com/libp2p/go-libp2p-quic-transport v0.2.2
-	github.com/libp2p/go-tcp-transport v0.1.1
-	github.com/libp2p/go-ws-transport v0.2.0
-	github.com/multiformats/go-multiaddr v0.2.0
-	github.com/multiformats/go-multistream v0.1.0
+	github.com/libp2p/go-libp2p-core v0.5.1
+	github.com/libp2p/go-libp2p-peerstore v0.2.3
+	github.com/libp2p/go-libp2p-quic-transport v0.3.3
+	github.com/libp2p/go-tcp-transport v0.2.0
+	github.com/libp2p/go-ws-transport v0.3.0
+	github.com/multiformats/go-multiaddr v0.2.1
+	github.com/multiformats/go-multistream v0.1.1
 	github.com/opentracing/opentracing-go v1.1.0
-	github.com/prometheus/client_golang v1.3.0
-	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/cobra v0.0.5
-	github.com/spf13/viper v1.6.2
-	github.com/syndtr/goleveldb v1.0.1-0.20190923125748-758128399b1d
+	github.com/prometheus/client_golang v1.5.1
+	github.com/sirupsen/logrus v1.5.0
+	github.com/spf13/cobra v0.0.7
+	github.com/spf13/viper v1.6.3
+	github.com/syndtr/goleveldb v1.0.0
 	github.com/uber/jaeger-client-go v2.22.1+incompatible
 	github.com/uber/jaeger-lib v2.2.0+incompatible // indirect
-	golang.org/x/crypto v0.0.0-20200311171314-f7b00557c8c4
-	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	resenje.org/web v0.4.0
+	golang.org/x/crypto v0.0.0-20200406173513-056763e48d71
+	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
+	resenje.org/web v0.4.3
```

Comment:
Can we leave unrelated dependencies unchanged in this pr? To restore them to the ones from master. I don't think that we should mix updating all dependencies with adding a new package that do not depend on them.